test/snippet/argument_parser/string_convertible.cpp

```diff
@@ -0,0 +1,46 @@
+#include <system_error>
+#include <seqan3/argument_parser/all.hpp>
+
+namespace seqan3::custom
+{
+// specialise the seqan3::custom::string_convertiblity data structure to enable parsing of std::errc.
+template <>
+struct string_convertiblity<std::errc>
+{
+    // specialise a mapping from your values of type Foo to the respective string
```

Comment:
```suggestion
    // Specialise a mapping from your values of type Foo to the respective string.
```

And also other way around?